torch/csrc/jit/codegen/cuda/disjoint_set.h

```diff
@@ -166,6 +166,20 @@ class DisjointSets {
  public:
   DisjointSets() = default;
 
+  DisjointSets(const DisjointSets<T, Hash>& other);
+
+  DisjointSets(DisjointSets<T, Hash>&& other);
+
+  DisjointSets<T, Hash>& operator=(const DisjointSets<T, Hash>& other);
+
+  DisjointSets<T, Hash>& operator=(DisjointSets<T, Hash>&& other);
```

Comment:
Can we make it like this?
```suggestion
  DisjointSets<T, Hash>& operator=(DisjointSets<T, Hash>&& other) = default;
```